src/flux/task-plan.coffee

```diff
@@ -459,6 +463,8 @@ TaskPlanConfig =
 
     isStatsFailed: (id) -> !! @_stats[id]
 
+    hasChanged: (id) -> not _.isEqual(@exports.getChanged.call(@, id), @_local[id].defaultPlan)
```

Comment:
You're right, it should be fairly easy to change.  I already updated the ones in plan-mixin.cjsx to use hasChanged, and the one in builder.cjsx is old code, so can be removed.  

I think that just leaves confirm-leave-mixin.cjsx, which is only used by the task plans, so I think I can change it there as well.  It might be a good idea to move that file into the task-plan folder, since that's the only place where it is included.